proposals/dictionary-expressions.md

```diff
@@ -206,21 +206,14 @@ An implicit *collection expression conversion* exists from a collection expressi
 *Collection expression conversions* require implicit conversions for each element.
 The element conversion rules are **updated** as follows.
 
-The implicit conversion exists if for each *element* `Eᵢ` in the collection expression:
-* **If the target *iteration type* is `KeyValuePair<K, V>`, then one of the following holds:**
-  * **If `Eᵢ` is a *key-value pair element* `Kᵢ:Vᵢ`, there is an implicit conversion from `Kᵢ` to `K` and an implicit conversion from `Vᵢ` to `V`.**
-  * **If `Eᵢ` is an *expression element* then one of the following holds:**
-    * **There is an implicit conversion from `Eᵢ` to `KeyValuePair<K:V>` where the conversion is one of:**
-      * ***default literal conversion***
-      * ***target-typed new conversion***
-    * **`Eᵢ` has type `KeyValuePair<Kᵢ:Vᵢ>` and there is an implicit conversion from `Kᵢ` to `K` and an implicit conversion from `Vᵢ` to `V`.**
-  * **If `Eᵢ` is a *spread element* `..Sᵢ`, where the *iteration type* of `Sᵢ` is `KeyValuePair<Kᵢ:Vᵢ>`, there is an implicit conversion from `Kᵢ` to `K` and an implicit conversion from `Vᵢ` to `V`.**
-
-* **If the target *iteration type* `T` *is not* `KeyValuePair<K, V>`, then one of the following holds:**
-  * If `Eᵢ` is an *expression element*, there is an implicit conversion from `Eᵢ` to `T`.
-  * If `Eᵢ` is a *spread element* `..Sᵢ`, there is an implicit conversion from the *iteration type* of `Sᵢ` to `T`.
-
-* **Otherwise there is *no conversion* from the collection expression to the target type.**
+The implicit conversion exists if the type has an *element type* `T` where for each *element* `Eᵢ` in the collection expression:
+* If `Eᵢ` is an *expression element* then:
+  * There is an implicit conversion from `Eᵢ` to `T`, **or**
+  * **There is no implicit conversion from `Eᵢ` to `T`, and `T` is a type `KeyValuePair<K, V>`, and `Eᵢ` has a type `KeyValuePair<Kᵢ, Vᵢ>`, and there is an implicit conversion from `Kᵢ` to `K` and an implicit conversion from `Vᵢ` to `V`.**
```

Comment:
I'm having trouble remembering where we landed with interest in expanding this. I'm interested in rules where there is an implicit conversion from `Eᵢ` to exactly one type `KeyValuePair<K, V>` where there is an implicit conversion from `Kᵢ` to `K` and an implicit conversion from `Vᵢ` to `V`.